### DISCUSSION TITLE
Support for Multiple image Upload Via Cloudinary

### DISCUSSION THREAD

**Discussion by gilbertlucas46** [keystonejs/keystone#6436] [Feature requests]
Right now there's not support for multiple image upload.

Checked the slack channel and I'm not the only one asking for it.
It's a basic feature to have.

![image](https://user-images.githubusercontent.com/5719346/131435578-6a6f22bb-05b1-40fa-a402-e0920320d8e7.png)

CreateInline just replaces the old image

**Comment by xmarkclx** [comment]
Multiple attachments upload would be nice as well, and would be good if it can also go to S3/Azure Storage etc.
